website/docs/api/indexing.mdx

```diff
@@ -25,7 +25,7 @@ These function are used for finding the H3 cell index containing coordinates, an
 <TabItem value="c">
 
 ```c
-H3Error pointToCell(const GeoPoint *g, int res, H3Index *out);
+H3Error latLngToCell(const LatLng *g, int res, H3Index *out);
```

Comment:
missing update to Python here

Reply:
I was holding off as we haven't made the Python changes yet, but I can put a placeholder here.